src/components/tableOfContents/js/TableOfContents.js

```diff
@@ -20,25 +20,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     *******/
     fluid.registerNamespace("fluid.tableOfContents");
 
-
-    fluid.tableOfContents.insertAnchor = function (name, element, anchorClass) {
-       // In order to resolve FLUID-4453, we need to make sure that the owner document is correctly
-       // taken from the target element (the preview may be in an iframe)
-        var anchor = $("<a></a>", element.ownerDocument);
```

Comment:
Is this comment above regarding iframe something that we need to be concerned with? I'm thinking not, as we are adding an id attribute to an existing element rather than adding a new element but wanted to check with you.